api/src/main/java/net/kyori/adventure/text/format/TextColor.java

```diff
@@ -47,6 +47,19 @@
  * @since 4.0.0
  */
 public interface TextColor extends Comparable<TextColor>, Examinable, RGBLike, StyleBuilderApplicable, TextFormat {
+  /**
+   * The hex character.
+   *
+   * @since 4.14.0
+   */
+  char SCREAMING_SNAKE_CASE1 = '#';
```

Comment:
maybe `HEX_CHARACTER` and `HEX_CHARACTER_BOXED`? idk

Reply:
I think the name should somehow indicate that it's used for colors. `TextColor.HEX_CHARACTER` feels very wrong, like, why it's even there.
As such color specifiers are often referred as "color codes", maybe something like `COLOR_CODE_PREFIX`?

Reply:
I mean, it is in `TextColor` -- maybe `HEX_PREFIX` though?